opentree/taxonomy_helpers.py

```diff
@@ -24,7 +24,7 @@ def download_taxonomy_file(version, loc):
         return path
     if os.path.exists(path + '.tgz') == False: #Download the file if you don't have it
         os.system('wget "http://files.opentreeoflife.org/ott/ott{}/ott{}.tgz" -P {}'.format(version, version, loc))
-    os.system("tar -xzvf {}/ott3.2.tgz -C {}".format(loc, loc))
```

Comment:
This is the only bug fix in this PR. The other changes are just minor or tools for debugging